### PR TITLE
Move CI to GitHub Actions

### PR DESCRIPTION
**Pull request by hugovk**
Fixes https://github.com/python-llfuse/python-llfuse/issues/47.

Sample build:

* https://github.com/hugovk/python-llfuse/actions/runs/421590705

Note that, unlike Travis CI, GitHub Actions doesn't have ppc64le.

### CODE REVIEW

**Comment by ThomasWaldmann** [comment]
the github actions stuff looks good, thanks!

how about just keeping the travis-ci config for the sake of ppc64le?

**Comment by hugovk** [comment]
Updated to keep Travis for ppc64le.

If you later find it takes too long or uses up too much credits, one idea is to test the lower and upper Python versions, or just a single version.

**Comment by ThomasWaldmann** [comment]
PR looks good, but needs a rebase onto current master to fix the conflicts.

**Comment by hugovk** [comment]
Rebased!

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install Linux dependencies
+        run: |
+          sudo apt-get install libattr1-dev
+          sudo apt-get install pkg-config
+          sudo apt-get install gcc
+          sudo apt-get install libfuse-dev
```
Comment (ThomasWaldmann) [inline .github/workflows/test.yml:30]: hmm, as these packages depend on other ones, doesn't this interactively ask and one should rather use `sudo apt-get install -y`?

Comment (ThomasWaldmann) [reply .github/workflows/test.yml:30]: maybe it also worked without `-y` but guess that is then only the case if all other packages are already present, which we can not really rely on.

**Review by ThomasWaldmann — COMMENTED** [review]
one minor nitpick...

File: .github/workflows/test.yml (hunk 1 above)
Comment (hugovk) [reply .github/workflows/test.yml]: I added `-y` to be on the safe side.

**Comment by ThomasWaldmann** [comment]
Thanks!

**Pull request merged**